Optional tag enforcement (#152)
schubergphilis/terraform-aws-mcaf-landing-zone: UPGRADING.md

# Upgrading to 0.19.x

> Be aware that all Tag policies are recreated since they are now created per tag_key in stead of per ou.

Reviewer: ```suggestion
Be aware that all tag policies will be recreated since they are now created per tag policy instead of per OU.
```
